documentation/wiki/Providing-Binary-Logs.md

```diff
@@ -6,23 +6,37 @@ However, you should be aware what type of information is captured in the binary
 
 ⚠ NOTE: some build environments make secrets available using environment variables. Before sharing a binary log, make sure it does not expose API tokens or other important secrets.
 
+## Capturing Binary Logs for command-line builds
```

Comment:
Do you want to add "preferred way" or "preferred way for command-line builds" to this header?

Reply:
That seems a bit much to me, but I pushed a change to try to both explain why VS is different and also emphasize the command-line option. Look ok?

Reply:
I'm a bit concerned someone might read that and think binlogs from VS aren't trustworthy at all. If something only reproduces from VS, I can tell it must be doing something different from a command line build...but what? How is it different? When does it build that the command line doesn't or vice versa? I think I'd be ok if I had some assurance that building from within VS isn't wrong or if I didn't see that at all, but I think this version raises more questions than it answers, personally.

Reply:
ok